R/zchunk_L222.en_transformation.R

```diff
@@ -1,6 +1,8 @@
 #' module_energy_L222.en_transformation
 #'
-#' Briefly describe what this chunk does.
+#' This chunk prepares the assumptions and calibrated outputs for energy transformation supplysectors, subsectors, and technologies.
```

Comment:
This should be brief (single sentence). Put the rest in `@details` below, which I see you haven't filled out. (Note - I'll do this as am rebuilding docs.)